### PR TITLE
1469 nvme wwn matching release 1.11.0 (release-1.11.0)

### PR DESCRIPTION
**Pull request by donatwork**
# Description
Doc changes to reflect issue 1469.

# GitHub Issues

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1469 |

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [X] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Comment by donatwork** [comment]
Closing to restart checks.

**Comment by donatwork** [comment]
Reopening to trigger checks.

File: content/v1/deployment/helm/drivers/upgrade/powerstore.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ weight: 1
 Description: Upgrade PowerStore CSI driver
 ---
 
+{{% pageinfo color="primary" %}}
+An issue impacting usage of the NVMe protocol has been identified in the PowerStore CSI Driver version v2.11.0. The problem prevents mounts of NVMe volumes across all orchestration platforms. Users are advised to use an alternate block protocol (e.g. iSCSI, FC) or delay upgrading to the v2.11.0 version of the PowerStore driver until an updated driver has been published.
```
Comment (sharmilarama) [inline content/v1/deployment/helm/drivers/upgrade/powerstore.md:11]: This upgrade page is in the context of upgrading 2.9 to 2.10.1, I think we can leave out the banner for 2.11 here.

Comment (donatwork) [reply content/v1/deployment/helm/drivers/upgrade/powerstore.md]: Removed.

**Review by anandrajak1 — APPROVED** [review]

**Review by gallacher — APPROVED** [review]

**Pull request merged**